packages/features/bookings/components/FiltersContainer.tsx

```diff
@@ -29,7 +29,7 @@ const filters: Filter[] = [
 
 export function FiltersContainer() {
   return (
-    <div className="flex w-full space-x-2 overflow-x-scroll rtl:space-x-reverse lg:justify-end ">
```

Comment:
had to use ml-auto instead of justify-end. justify-end did not work when filters were overflowing